montecarlo/config/SR1_parameters.ini

```diff
@@ -0,0 +1,315 @@
+[pax]
+parent_configuration = "pmt_afterpulsing"  # (should be checked for SR1)
+look_for_config_in_runs_db = False
+
+
+[WaveformSimulator]
+
+# S2 electron drift and extraction
+diffusion_constant_liquid =           31.73 * cm**2 / s  # Cathode at -8 kV, xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width
+elr_gas_gap_length =                  2.66 * mm          # Average of level meters in run 3410 (should be checked for SR1)
+s2_secondary_sc_gain =                23                 # photons/SE, xenon:xenon1t:aalbers:single_electron_gain_stability
+                                                         # (should be checked for SR1)
+# Distance from anode where field becomes wire-like (~1/r) rather than uniform:
+anode_field_domination_distance =     0.372 * mm         # xenon:xenon1t:aalbers:single_electron_waveform_shape (should be checked for SR1)
+triplet_lifetime_gas      =           112.6              # +- 9.6 ns. xenon:xenon1t:aalbers:single_electron_waveform_shape
+                                                         # (should be checked for SR1)
+gas_gap_warping_map =                 'gas_gap_warping_map_Oct16.pkl'   # Fit to single electrons (should be checked for SR1)
+
+# Used for s1 time structure calculations (currently not used; actually using empirical model):
+drift_field =                         82 * V /cm        
+
+# Light distribution
+s1_light_yield_map =                  's1_xyz_XENON1T_kr83m-sr1_pax-664_fdc-adcorrtpf.json'
+s2_light_yield_map =                  's2_xy_map_v2.1.json'
+s2_fitted_patterns_file =             'XENON1T_s2_xy_fitted_patterns_top_v0.3.0.json.gz'
+s2_mean_area_fraction_top =           0.627    # See xenon:xenon1t:sim:notes:yuehuan:sr_1_kr83m&#s2_top_fraction
+
+# Various data-extracted corrections
+rz_position_distortion_map =          'FDC-SR1_AdCorrTPF.json.gz'
+
+
+[NeuralNet.PosRecNeuralNet]
+neural_net_file = 'NN_XENON1T_v8_mc_v030_sr1_n15.npz'
+
+[MC]
+mc_generated_data = True
+mc_sr_parameters = "sr1"
+mc_run_number = 10000
+
+[DEFAULT]
+
+electron_lifetime_liquid =      550 * us           # xenon:xenon1t:org:commissioning:meetings:20170628#electron_lifetime
+drift_velocity_liquid =         1.335 * um / ns    # Cathode at 8 kV, xenon:xenon1t:yuehuan:analysis:1sciencerun_s2width
+                                                   # and https://github.com/XENON1T/lax/pull/62
+
+# PMT gains
+# The correct TPC PMT gains are now fetched from the runs db, these are just some placeholder values
+# Only for waveform simulation usage
+# @ 2017-09-05 Switch to use the PMT mask for SR1
+# from the note:
+# xenon:xenon1t:analysis:sciencerun1:pmt
+# gain values from:
+# https://xe1t-wiki.lngs.infn.it/lib/exe/fetch.php?media=xenon:gain_sr1_20170815.html
+gains = [
```

Comment:
I **_believe_** that [self-trigger thresholds](https://github.com/XENON1T/pax/blob/master/pax/config/XENON1T.ini#L193) have also changed, and perhaps need to be updated for simulation?

Reply:
I'm not familiar with what you're saying.  Can you clarify what exactly you think needs to be changed?

These gains were already updated by @mcfatelin coming from @skazama.

And does "self-trigger threshold" already get counted in @ershockley's SPE model?

Reply:
Sorry I don't quite follow the discussion, but the SPE shapes come from LED data and so don't contain any information about the thresholds.